development_setup/resource/volumes/hub/backups/sdb-test.bak

```diff
@@ -0,0 +1,31 @@
+-- ------------------------------
```

Comment:
Do we need these bak files in github?
or can we filter by gitignore?

Reply:
All the empty bin/* files are for future implementation, so we within the containers can run the commands like `bin/start` for starting application or server

There're files which HAVE been filled with code in case of the SurrealDB
like `development_setup/resource/bin/pwa/sql`.

The douplication can be illiminated

Reply:
The Backups CAN be ignored, if need be. (they're as of now only used in the db containers)
But for now i think it would be nice to be able to share db setup